client/index.html

```diff
@@ -60,6 +60,10 @@ <h2 class="title">The Lounge is loading…</h2>
 					</div>
 				</div>
 				<div id="sign-in" class="window">
+					<div class="header">
+						<span class="title">The Lounge</span>
+						<span class="topic">Web IRC Client</span>
```

Comment:
Well, there is no sign in page on a public instance 😄.

Anyway, I'm happy with "The Lounge", but "Web IRC Client" or any description is really unnecessary here.
What do other @thelounge/maintainers think?